src/mbgl/renderer/renderer_impl.cpp

```diff
@@ -322,7 +323,8 @@ void Renderer::Impl::render(const RenderTree& renderTree) {
 #endif
 
     // Give the layers a chance to do cleanup
-    orchestrator.observeLayerGroups([&](LayerGroup& layerGroup) { layerGroup.postRender(orchestrator, parameters); });
```

Comment:
hm, this needs to be reverted

Reply:
Was it removed intentionally or just a merge issue or something?  I don't think anything is actually using that yet, it's just speculative.

Reply:
just a line commented for debug purposes that got away.
we're not doing anything in postRender yet afaik